components/Partners/Partners.js

```diff
@@ -0,0 +1,15 @@
+import Section from '../Section';
+import styles from './Partners.scss';
+
+const Partners = () => (
+  <Section className={styles.partners}>
+    <h3>Our Partners</h3>
+    <div className={styles.logos}>
```

Comment:
![image](https://user-images.githubusercontent.com/31391056/46947778-452e8980-d07c-11e8-90a0-4dc2df6acd1d.png)
This might be psychological but it doesn't look like the items are perfectly centered in comparison to the "our partners" heading

Reply:
I think this depends on the SVG for NEX being the same with as the SVG for NGC.  /cc @fabric-8